docs/scripts.rst

```diff
@@ -0,0 +1,194 @@
+..
+  SPDX-FileCopyrightText: 2022 Nico Rikken <nico.rikken@fsfe.org>
+
+  SPDX-License-Identifier: CC-BY-SA-4.0
+
+==============
+Helper scripts
+==============
+
+This section contains scripts and snippets to help with the usage of reuse.
+Reuse already has a lot of functionality built-in, but there are still cases where reuse could use support from some external scripts.
```

Comment:
Can we settle on one way to write REUSE? I'd opt for REUSE generally, or "REUSE helper tool" when we speak specifically about the tool.

Reply:
Great, I was looking for guidance on it while writing. i changed it.